script/upgrades/MU03/MU03Checks.sol

```diff
@@ -134,6 +142,21 @@ contract MU03Checks is Script, Test {
     console2.log("\tBiPoolManagerProxy has a correct implementation address 🫡");
   }
 
+  function verifySortedOracles() internal view {
+    SortedOraclesProxy soProxy = SortedOraclesProxy(sortedOraclesProxy);
```

Comment:
This is correct but I kinda want us to move away from using named proxies so for the sake of that I'd import the base `Proxy` contract instead and cast it as that here, just to future-proof the system.